pkg/kubelet/pleg/generic.go

```diff
@@ -445,10 +444,23 @@ func updateRunningPodAndContainerMetrics(pods []*kubecontainer.Pod) {
 			// update the corresponding "container_state" in map to set value for the gaugeVec metrics
 			containerStateCount[string(container.State)]++
 		}
+
+		sandboxes := pod.Sandboxes
```

Comment:
just noting how this regressed:

https://github.com/kubernetes/kubernetes/commit/c02d49d775b4dc960f52af1f5295642c07947ca7

moved a metric that was sourced from the kubelet runtime cache to now be sourced by the pleg directly.